modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -91,15 +97,15 @@ spec:
         - tls://kafka-broker1.example.com:9093/
         - tls://kafka-broker2.example.com:9093/
       topic: app-topic <3>
-...
+# ...
 ----
 <1> Specify a `kafka` key that has a `brokers` and `topic` key.
 <2> Use the `brokers` key to specify an array of one or more brokers.
-<3> Use the `topic` key to specify the target topic that will receive the logs.
+<3> Use the `topic` key to specify the target topic that receives the logs.
 
-. Create the CR object:
+. Apply the `ClusterLogForwarder` CR by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc apply -f <filename>.yaml
```

Comment:
Would this need a callout? Someting like: "Replace `<filename>` with the name of the file." Not sure. Maybe it's already obvious, but the other steps in the proc use callouts. Just a thought for you to consider...

Reply:
I don't think so. This is a command that's used really widely across products in our docs and it's never used with a callout 🙂